### ISSUE TITLE
IMGUI 2.53 update - Unable to scroll inventory using scroll wheel if secondary slot is occupied.

### ISSUE THREAD

**Issue by WCPistolPete** [SmartlyDressedGames/Unturned-3.x-Community#2241]
.253 update*. Scroll wheel works perfectly as long as the secondary slot is empty. If the slot is occupied then one must use the cursor and left mouse button to drag the scroll bar. Empty the secondary slot, back to normal.

**Comment by SDGNelson** [comment]
Thanks for the report! Good catch finding that those were related. This was a very confusing bug to track down this morning, as obviously there is no overlap between them and should not be affecting one another. It should be fixed for the update today.

**Comment by WCPistolPete** [comment]
It was quite the discovery. I couldn't scroll on my single-player PEI map, but I could on Washington. Reset PEI, all good. Started loading up items. Tested scrolling after adding a top, bottom, beret, vest, NVG, pack, primary and then... The last slot I filled was the secondary. :P I played around just swapping different items into and out of the secondary just to see if one item (PDW is my usual secondary) caused the bug or if any item could cause the bug. It made me chuckle, but then I thought about the reaction if the fix didn't make it into the patch in time. Glad to see the fix made it into the patch.
